src/actionCreators/languages.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import { loadingLanguages, languagesReceived } from 'actions/entities'
+
+const loadLanguages = () => (dispatch, getState) => {
+  if (getState().selectorReducer.entities.languages) {
+    return // Languages already loaded
+  }
+
+  dispatch(loadingLanguages())
+  fetch('https://id.loc.gov/vocabulary/iso639-1.json')
+    .then(resp => resp.json())
+    .then((json) => {
+      dispatch(languagesReceived(json))
+    })
+    .catch(() => false)
```

Comment:
Log the error?

Reply:
This is just moving this code here. This is exactly how it was written when it was in https://github.com/LD4P/sinopia_editor/pull/1012/files#diff-2435c181d99c6352e3e1322a7ce3eb43L85